notebooks/407-pedestrian-tracking-webcam/407-pedestrain-tracking.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.        img_batch = np.concatenate([
</pre><p>probably you can reuse preprocess image here, just stay concatenation of preprocessed images here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>